projects/graph/src/old_draw.py

```diff
@@ -0,0 +1,77 @@
+"""
+General drawing methods for graphs using Bokeh.
+"""
+import math
+
+from bokeh.io import show, output_file
+from bokeh.plotting import figure
+from bokeh.models import (
+    GraphRenderer,
+    StaticLayoutProvider,
+    Oval,
+    LabelSet,
+    ColumnDataSource,
+)
+from bokeh.palettes import Spectral4
```

Comment:
Nice choice of colors for your palette. They really help your graphs stick out and visually pop.